docker/Dockerfile

```diff
@@ -78,7 +78,7 @@ FROM builder AS tester
 COPY --from=us-docker.pkg.dev/zealous-zebra/zebra/zcash-params /root/.zcash-params /root/.zcash-params
 COPY --from=us-docker.pkg.dev/zealous-zebra/zebra/lightwalletd /lightwalletd /usr/local/bin
 
-RUN cargo test --locked --release --features enable-sentry --workspace --no-run
+RUN cargo test --locked --release --features enable-sentry --no-run
```

Comment:
Question: why is this change in this PR? What does it do?

Reply:
Fixed by: [fix: revert uneeded changes](https://github.com/ZcashFoundation/zebra/pull/4252/commits/f516ae5b636a84b20f70444847fe86ef8b6d551b)